setup.cfg

```
@@ -3,9 +3,9 @@ name = kaitaistruct
 version = attr: kaitaistruct.__version__
 author = Kaitai Project
 author_email = greycat@kaitai.io
-url = http://kaitai.io
+url = https://kaitai.io
 description = Kaitai Struct declarative parser generator for binary data: runtime library for Python
-long_description = file: README.rst
+long_description = file: README.md
```

Just noticed that we also need to set `long_description_content_type`, otherwise it defaults to `text/x-rst`, which is not correct. (`twine check` warns about this.)

```suggestion
long_description = file: README.md
long_description_content_type = text/markdown
```